### PR TITLE
Add median()  to ndframe

### PR DESCRIPTION
**Pull request by mesejo**
Includes:
- Add implementation of median
- Add docstring of std, var and, median
- Run black

### CODE REVIEW

**Comment by sethmlarson** [comment]
Jenkins test this please

File: eland/ndframe.py [hunk 1]
```diff
@@ -252,11 +252,95 @@ def min(self, numeric_only=True):
...
+            The value of the variance for each numeric column
+
+        See Also
+        --------
+        :pandas_api_docs:`pandas.DataFrame.var`
+
+        Examples
+        --------
+        >>> df = ed.DataFrame('localhost', 'flights')
+        >>> df.std() # doctest: +SKIP
```
Comment (sethmlarson) [inline eland/ndframe.py:270]: Should be .var()?

Comment (mesejo) [reply eland/ndframe.py]: Done

**Comment by mesejo** [comment]
@sethmlarson Regarding the refactoring, here are my thoughts:

At least `aggs`, `_metric_aggs`, and `describe` share functionality. All of the functions built aggregation queries, perform some operations, and extract the information from the aggregation. My suggestion is to extract the built aggregation and extraction functionalities from those functions into separate functions (some candidates): `built_agg_query` and `extract_agg_response` and refactor the above mentioned functions.

**Comment by sethmlarson** [comment]
@mesejo Would be good to think about `terms_aggs()` and `hist_aggs()` as well, I like the idea of build_X_query() and extract_X_response(). Would be good on the `extract_X_response()` to have the return type be something that can be returned into a `pd.DataFrame()` or `pd.Series()` so it can be used within `agg()` which returns `DataFrame` and the individual aggs like `median()` that return a `Series`.

**Comment by mesejo** [comment]
@sethmlarson I'm willing to do it, but I cannot guarantee that I can deliver it fast

**Pull request merged**